src/abstractdataframe/show.jl

```diff
@@ -24,12 +24,24 @@ end
 Render a value to an IO object. Unlike
 `show`, render strings without surrounding quote marks.
 """
-ourshow(io::IO, x::Any) =
-    show(IOContext(io, :compact=>get(io, :compact, true), :typeinfo=>typeof(x)), x)
+function ourshow(io::IO, x::Any)
+    ctxt = IOContext(io, :compact=>get(io, :compact, true), :typeinfo=>typeof(x))
```

Comment:
Thanks. Matrix printing in Base does something a bit more tricky (and which I find a bit ugly, but well...):
https://github.com/JuliaLang/julia/pull/34387/files#diff-6f3c3cea343e12203f3635ea6504d6b3R105
We should probably do the same.

Reply:
I've updated the code to use the same behaviour